### PR TITLE
Add test for null error message

### LINKED ISSUE #5: Add test for empty original error message
Observed missing in code coverage

### PR DESCRIPTION
**Pull request by shaishavgandhi05**
Resolves #5

### CODE REVIEW

File: rxdogtag/src/test/java/com/uber/anotherpackage/DogTagObserverTest.java [hunk 1]
```diff
@@ -120,6 +120,28 @@ public void observerWithErrorHandling_ignoredByRxDogTag() {
...
   }
 
+  @Test
+  public void nullOriginErrorMessage_replacedWithEmptyString() {
+    Exception originalError = new IllegalStateException();
+    int expectedLineNumber = subscribeError(Observable.error(originalError));
+
+    Throwable e = errorsRule.take();
+    assertThat(e).isInstanceOf(OnErrorNotImplementedException.class);
+    // Original error message was null. Now replaced by empty string
```
Comment (ZacSweers) [inline rxdogtag/src/test/java/com/uber/anotherpackage/DogTagObserverTest.java]: I wonder if we should put something more descriptive here, like `"<no error message>"`.
